package.json

```diff
@@ -42,7 +42,7 @@
     "clean": "rm -rf build docs/static/css node_modules/ yarn-error.log",
     "percy": "percy exec -- node snapshots.js"
   },
-  "version": "2.19.3",
+  "version": "2.20.0",
```

Comment:
Oh, sorry, I meant updating the version in component status page "What's new in Vanilla 2.20" heading, not in package.json yet - lets leave it for the release.